src/native/fs/fs_napi.cpp

```diff
@@ -2163,16 +2202,22 @@ fs_napi(Napi::Env env, Napi::Object exports)
                 PANIC("Error: %s\n"
                     << uv_dlerror(lib));
             }
+            if (uv_dlsym(lib, "gpfs_ganesha", (void**)&dlsym_gpfs_ganesha)) {
+                PANIC("Error: %s\n"
+                    << uv_dlerror(lib));
+            }
             if (sizeof(struct gpfsRequest_t) != 256) {
                 PANIC("The gpfs get extended attributes is of wrong size" << sizeof(struct gpfsRequest_t));
             }
+            
             auto gpfs = Napi::Object::New(env);
             // for now we export an (empty) object, which can be checked to indicate that
             // gpfs lib was loaded and its api's can be used.
             // e.g: gpfs["version"] = Napi::String::New(env,  gpfs_get_version());
             // e.g: gpfs["foo"] = Napi::Function::New(env, api<Foo>);
             // gpfs.Freeze();
             exports_fs["gpfs"] = gpfs;
+            exports_fs["register_gpfs_noobaa"] = Napi::Function::New(env, register_gpfs_noobaa);
```

Comment:
I think it would be nicer to put this on the gpfs object -
```suggestion
            gpfs["register_gpfs_noobaa"] = Napi::Function::New(env, register_gpfs_noobaa);
```